devel/ansible/roles/dev/tasks/main.yml

```diff
@@ -35,6 +36,11 @@
       - redhat-rpm-config
       - zlib-devel
 
+- name: Install sqlalchemy_schemadisplay
+  command: pip install sqlalchemy_schemadisplay
+  args:
+      creates: /usr/lib/python2.7/site-packages/sqlalchemy_schemadisplay.py
```

Comment:
@jeremycline Can you move this to be installed by RPM in the list above, with the assumption that it will be called ```python2-sqlalchemy_schemadisplay```?